api.php

```diff
@@ -1173,6 +1173,10 @@ public function run($userID, $permissionIsExplicit) {
 
 		$timeSent = libGameMessage::send($toCountryID, $countryID, $message);
 
+		// I cannot put this require in the first lines of the page, it gives me an error
+		require_once('lib/pusher.php');
```

Comment:
I see, so this triggers the pusher to send messages? Very cool!

I think you need to move this inside of `libGameMessage::send` to catch messages sent from places other than the beta UI. Even then, I'm not sure it catches everything... you should grep for inserts into `wD_Messages` to see if there are any other places.

Reply:
It looks to me like libGameMessage is the central point for message sending, a bit of grepping didn't obviously turn up other places where it gets modified.